### PR TITLE
DOCS-582: Mark robot service page as draft

### PR DESCRIPTION
**Pull request by sguequierre**
* make it a draft it case we want to redocument these methods soon

### CODE REVIEW

File: docs/viam/_index.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ This provides flexibility and security whether you are building tight control lo
 
 There are four categories of APIs:
 
-- [Robot](/services/robot-service/) - high level robot commands
+- [Robot](https://github.com/viamrobotics/rdk/blob/main/robot/server/server.go) provides high level robot commands
```
[the remark this answers is not in the document]
Comment (edaniels) [reply docs/viam/_index.md:92]: this shouldn't for the same reason I mentioned in the other comment

**Review by edaniels — CHANGES_REQUESTED** [review]
See new comment

File: docs/viam/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/viam/_index.md:92]: @edaniels Sorry for the confusion, what do you mean? Shouldn't link to /server go here? Should I remove the bullet entirely?

[the remark this answers is not in the document]
Comment (npentrel) [reply docs/viam/_index.md:92]: Leaning towards removing the bullet point for now. But @edaniels can decide.

Comment (sguequierre) [inline docs/viam/_index.md:92]:
```suggestion
```

File: docs/viam/_index.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ This provides flexibility and security whether you are building tight control lo
 
 There are four categories of APIs:
```
Comment (sguequierre) [inline docs/viam/_index.md:90]:
```suggestion
There are three categories of APIs:
```

File: docs/viam/_index.md (hunk 1 above)
Comment (sguequierre) [reply docs/viam/_index.md:92]: Github is not working

[the remark this answers is not in the document]
Comment (edaniels) [reply docs/viam/_index.md:92]: We should link to our docs on robot proto

**Review by edaniels — CHANGES_REQUESTED** [review]

File: docs/viam/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/viam/_index.md:92]: Do you mean https://pkg.go.dev/go.viam.com/rdk@v0.2.31/robot ? I don't think we have any others.

[the remark this answers is not in the document]
Comment (edaniels) [reply docs/viam/_index.md:92]: There's https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto.

[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/viam/_index.md:92]: @edaniels Have now made this revision in https://github.com/viamrobotics/docs/pull/912 if you want to give it a look! This pr is left over on old fork from last week's change with the docs being made public, 912 is what can be merged

**Comment by edaniels** [comment]
we should update title of this PR

**Pull request closed without merging**